### ISSUE TITLE
Setting learning rate initialisation and schedule using a small subset of training data

### ISSUE THREAD

**Issue by matt-graham** [Neuroglycerin/neukrill-net-work#66]
[This book chapter](http://link.springer.com/chapter/10.1007/978-3-642-35289-8_25/fulltext.html#Fig6) suggests learning rate schedule and initialisation can be validly set using a smaller (representative) subset of the training data. We should therefore implement some way of specifying to use only a subset of data in run settings (e.g. specify ratio example to use over total available) to allow quicker experimentation to identify a good learning rate schedule and initialisation.
